openid-4-verifiable-presentations-1_0.md

```diff
@@ -460,6 +477,43 @@ To use `client_id_scheme` values `entity_id`, `did`, `verifier_attestation`, `x5
 
 Other specifications can define further values for the `client_id_scheme` parameter. It is RECOMMENDED to use collision-resistant names for such values.
 
+# Create Request Endpoint {#create_response_uri}
+
+This endpoint is offered by the Verifier. The Wallet sends a request to this endpoint if the Verifier requests so by passing the `create_request_uri` authorization request parameter. In case of success, the response is a request object that the Wallet MUST process in the same way as a request object as defined in [@RFC9101]. 
```

Comment:
Incosistent use of request object. Later on it is mentioned that this is the signed presentation request. IMO, the terminology should be made more clear. Especially, it is not clear what is the difference between the Authorization Request Object (as per JAR) and the new partial request object that this mechanism introduces.

Reply:
***Updated:***
This comment got superseded by the most recent changes and my other suggestions/comments below.